build.js

```diff
@@ -5,7 +5,7 @@ const yaml = require('yaml');
 // this means that yaml anchors will be "exploded" and not present in the github workflow
 // we write out, which means we can use anchors when developing flowzone but not have to
 // worry about github actions not supporting them
-const flowzone = yaml.parse(fs.readFileSync('./flowzone.yml', 'utf8'));
+const flowzone = yaml.parse(fs.readFileSync('./flowzone.yml', 'utf8'),{merge: true});
```

Comment:
The correctly formatted version (if we had balena-lint set up on this repo) would be:
```suggestion
const flowzone = yaml.parse(fs.readFileSync('./flowzone.yml', 'utf8'), {
	merge: true
});
```